pkgs/applications/misc/premid/default.nix

```diff
@@ -0,0 +1,86 @@
+{ autoPatchelfHook, makeDesktopItem, lib, stdenv, wrapGAppsHook, fetchurl
+, alsa-lib, at-spi2-atk, at-spi2-core, atk, cairo, cups, dbus, expat, fontconfig
+, freetype, gdk-pixbuf, glib, gtk3, libcxx, libdrm, libnotify, libpulseaudio, libuuid
+, libX11, libXScrnSaver, libXcomposite, libXcursor, libXdamage, libXext
+, libXfixes, libXi, libXrandr, libXrender, libXtst, libxcb, libxshmfence
+, mesa, nspr, nss, pango, systemd, libappindicator-gtk3, libdbusmenu
+}:
+
+let
+  name = "PreMiD";
+in stdenv.mkDerivation rec {
```

Comment:
Why do we need this variable?

Reply:
That's how app devs cased it

Reply:
I meant do we need that variable at all? I don't think it will change anytime soon and we could just hardcode the value everywhere it is used. It makes the code easier to read and understand and compare to upstreams instructions.